ThirdPartyKeys.example.plist

```diff
@@ -5,11 +5,11 @@
 	<key>FacebookKey</key>
 	<dict>
 		<key>Release</key>
-		<string></string>
+		<string>344767479509537</string>
```

Comment:
You should remove the `.example` from the file name because this file must not be in version control